srcpkgs/httpie/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'httpie'
 pkgname=httpie
-version=2.2.0
-revision=2
+version=2.5.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-requests python3-Pygments"
+depends="python3-setuptools python3-requests python3-requests-toolbelt
+ python3-Pygments python3-pysocks python3-defusedxml"
```

Comment:
We should be able to remove setuptools from runtime dependencies, it's used only for building.

Reply:
Actually it is used at runtime too.

Reply:
Out of curiosity, could you point me where it is imported? Searching through the source code only reveals it used in setup.py.

Reply:
Here: https://github.com/httpie/httpie/blob/e2ba214ac02905ab0ada72c582c9e95023c49496/httpie/plugins/manager.py#L5

FTR a similar problem was fixed on Alpine: https://github.com/httpie/httpie/issues/1092